docs/user-guide/configuration/externalConfig.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: main
+title: External Config
+category: User Guide
+menu: menu
+toc:
+    - title: External Config
+      url: "#external-config"
```

Comment:
you can actually also link to headers that are ##, but it should look like this:
```
toc:
    - title: External Config
       url: "#external-config"
    - title: Configure external config in parent pipeline
       url: "#configure-external-config-in-parent-pipeline"
```